### PR TITLE
Update fastlane UI handler to support FastlaneCI rendering

### PR DESCRIPTION
**Pull request by KrauseFx**
Related to https://github.com/fastlane/ci/pull/139

### CODE REVIEW

**Comment by KrauseFx** [comment]
Whops tests fail

**Review by taquitos — APPROVED** [review]
🎈:goat:

**Comment by KrauseFx** [comment]
Will fix the tests later on, as soon as the implementation fully works

**Comment by taquitos** [comment]
I couldn't help but notice.....

**Comment by KrauseFx** [comment]
@taquitos Ha, the `[WIP]` in the PR title? 😉

File: fastlane/spec/plugins_specs/plugin_manager_spec.rb [hunk 1]
```diff
@@ -106,19 +106,19 @@
         end
 
         it "supports specifying a custom local path" do
-          expect(FastlaneCore::UI.current).to receive(:select).and_return("Local Path")
-          expect(FastlaneCore::UI.current).to receive(:input).and_return("../yoo")
+          expect(FastlaneCore::UI.ui_object).to receive(:select).and_return("Local Path")
+          expect(FastlaneCore::UI.ui_object).to receive(:input).and_return("../yoo")
```
Comment (taquitos) [inline fastlane/spec/plugins_specs/plugin_manager_spec.rb]: OMG I LOVE TESTS

File: fastlane_core/spec/ui_spec.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+
+describe FastlaneCore::UI do
```
Comment (taquitos) [inline fastlane_core/spec/ui_spec.rb]: ❤️

**Comment by taquitos** [comment]
🎈🐐🎈🐐🎈🐐

**Comment by taquitos** [comment]
Whoops

**Pull request merged**